tests/test_saml2_client.py

```diff
@@ -1,15 +1,27 @@
+from datetime import timedelta
+
 from django.test import TestCase
 from django.urls import reverse
+from django.utils import timezone
 
 import pytest
 
 from digid_eherkenning.models import DigidConfiguration, EherkenningConfiguration
+from digid_eherkenning.models.certificates import ConfigCertificate
 from digid_eherkenning.saml2.digid import DigiDClient
 from digid_eherkenning.saml2.eherkenning import eHerkenningClient
 
 
 @pytest.mark.usefixtures("digid_config_defaults", "temp_private_root")
 class DigidClientTests(TestCase):
+
+    def setUp(self):
```

Comment:
always make sure to call the `super().setUp()` in case mixins get added in the future :)